ee_plugin/processing/add_image_collection.py

```diff
@@ -189,9 +190,21 @@ def update_percentile_visibility(index):
             title=_("Filter by Coordinates"),
             collapsed=True,
         )
+        self.extent_group.setMapCanvas(Map.get_iface().mapCanvas())
```

Comment:
@alukach; adding the set by current mapCanvas option turned out to be built into the extent group